nuntium/views.py

```diff
@@ -92,14 +96,21 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class MessageDetailView(DetailView):
-    template_name = 'nuntium/message/message_detail.html'
+class MessageDetailViewMixin(DetailView):
     model = Message
 
     def get_queryset(self):
```

Comment:
Uhh yeah, you're right! I hadn't seen that. Well there's something else related to the url cause what it does is include the instance_slug https://github.com/ciudadanointeligente/write-it/blob/198-add-page-post-submission/nuntium/subdomain_urls.py#L17, and that's why it was originally intended to use this get_queryset. But I could definately move this over to the get_object method.